tests/fixtures/fixture_exchange_collector.py

```diff
@@ -0,0 +1,20 @@
+import pytest_cases
+
+
+@pytest_cases.fixture(scope="function")
+def output_config_toml() -> str:
+    input_file_content = """
+                        [[general]]
+                        output_dir = "."
+
+                        [[exchanges]]
+                        
+                        [[exchanges.mf-ridv2dflow1d_flux_output]]
```

Comment:
How do we deal with writing errors in labels? the labels dont seem to correspond to the exchangelabels of toml file? Also we don't define "towards" and "from" fluxes in toml file.